src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/InviteUsers/ISendOrganizationInvitesCommand.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Bit.Core.AdminConsole.Entities;
+using Bit.Core.AdminConsole.OrganizationFeatures.OrganizationUsers.InviteUsers.Models;
+using Bit.Core.Entities;
+using Bit.Core.Models.Mail;
+
+namespace Bit.Core.AdminConsole.OrganizationFeatures.OrganizationUsers.InviteUsers;
+
+public interface ISendOrganizationInvitesCommand
+{
+    Task SendInvitesAsync(SendInvitesRequest request);
+
+    Task<OrganizationInvitesInfo> BuildOrganizationInvitesInfoAsync(IEnumerable<OrganizationUser> orgUsers, Organization organization, bool initOrganization = false);
```

Comment:
This is only used in [OrganizationService](https://github.com/bitwarden/server/blob/829e49c3980811d033da96184ed3ba65b748ab38/src/Core/AdminConsole/Services/Implementations/OrganizationService.cs#L1058-L1059), where it's immediately followed by a call to `mailService.SendOrganizationInviteEmailsAsync` - which is exactly what `ISendOrganizationInvitesCommand.SendInvitesAsync` does.

Could orgService call `ISendOrganizationInvitesCommand.SendInvitesAsync` instead, and make this one private?